<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профіль користувача</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Стилі можна скопіювати з mainwindow.ui або задати свої */
QWidget {
    font-family: &quot;Segoe UI&quot;, system-ui, -apple-system, BlinkMacSystemFont, sans-serif;
    font-size: 10pt;
}
QLabel#profileHeaderLabel {
   font-size: 16pt;
   font-weight: 600;
   color: #333;
   margin-bottom: 15px;
}
QLabel {
 color: #333;
}
QLabel[objectName^=&quot;label_&quot;] { /* Стиль для міток полів */
  font-weight: 500;
  color: #555;
}
QPushButton {
    padding: 8px 16px;
    background-color: #0078d4;
    color: white;
    border: 1px solid #0078d4;
    border-radius: 5px;
    font-weight: 500;
    min-height: 30px; /* Трохи менша висота для кнопок діалогу */
    font-size: 10pt;
}
QPushButton:hover {
    background-color: #106ebe;
    border-color: #106ebe;
}
QPushButton:pressed {
    background-color: #005a9e;
    border-color: #005a9e;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="profileHeaderLabel">
     <property name="text">
      <string>Ваш профіль</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="profileFormLayout">
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_FirstName">
       <property name="text">
        <string>Ім'я:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_LastName">
       <property name="text">
        <string>Прізвище:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Email">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Phone">
       <property name="text">
        <string>Телефон:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_Address">
       <property name="text">
        <string>Адреса:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_JoinDate">
       <property name="text">
        <string>Дата реєстрації:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="joinDateLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_Loyalty">
       <property name="text">
        <string>Програма лояльності:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="loyaltyLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_Points">
       <property name="text">
        <string>Бонусні бали:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="pointsLabel">
       <property name="text">
        <string>(завантаження...)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool> <!-- Змінено на true для центрування кнопки -->
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
